cluster/images/etcd-empty-dir-cleanup/etcd-empty-dir-cleanup.sh

```diff
@@ -17,7 +17,7 @@
 echo "Removing empty directories from etcd..."
 
 cleanup_empty_dirs () {
-  if [[ $(${ETCDCTL} ls $1) ]]; then
+  if [[ "$(${ETCDCTL} ls $1)" ]]; then
```

Comment:
I am not a sh/bash expert, but my understanding is `[[..]]` is built-in (either supported or not but if it is supported it is a built-in check) and does not require quotes. This make me wonder if this works as expect. of course if the if statement returns true all the time, the script would still remote empty folders, but then we can simplify it by removing if statement. Have you check this if statement in busybox's sh?

ref: http://serverfault.com/questions/52034/what-is-the-difference-between-double-and-single-square-brackets-in-bash